src/convexhull.jl

```diff
@@ -0,0 +1,31 @@
+"""
+    upperhull(pset)
+
+Upper part of the convex hull of a given set of 2D points `pset`. 
```

Comment:
I didn't have time to look into the modification of the original algorithm but  I had understood that it returned the complete hull, not just the upper part. Can you please clarify?

Reply:
I won't be able to keep working on this. It was a nice try though!

Reply:
Thanks for starting it out @Andre2957 , good luck with your qualification exam!